tests/integration/CartDiscount/CartDiscountCreateRequestTest.php

```diff
@@ -54,4 +54,30 @@ function (CartDiscount $cartDiscount) use ($client) {
             }
         );
     }
+
+    public function testDeleteByKey()
```

Comment:
for consistancy: should we change the place of this test? in other part of the code it's in createRequestTest or in Update. Or should we create a deleteRequestTest class?

Reply:
As already said in the other PR. not really necessary.